### PR TITLE
[stable/jenkins] configure agent.args in values.yaml

### PR DESCRIPTION
**Pull request by torstenwalter**
#### What this PR does / why we need it:

We have agent.args is empty by default, but then we set default when rendering templates. This is confusing to users and could be simplified.

- ./templates/_helpers.tpl

  ```
          {{- if .Values.agent.args }}
          args: "{{ .Values.agent.args }}"
          {{- else }}
          args: "^${computer.jnlpmac} ^${computer.name}"
          {{- end }}
  ```

- ./templates/config.yaml
  ```
  {{- if .Values.agent.args }}
                    <args>{{ .Values.agent.args }}</args>
  {{- else }}
                    <args>${computer.jnlpmac} ${computer.name}</args>
  {{- end }}
  ```

It would be best if we could set this here to:
```yaml
args: "${computer.jnlpmac} ${computer.name}"
```

and take care of escaping dollar signs in the template.

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/jenkins/templates/_helpers.tpl [hunk 1]
```diff
@@ -103,11 +103,7 @@ jenkins:
       templates:
       - containers:
         - alwaysPullImage: {{ .Values.agent.alwaysPullImage }}
-          {{- if .Values.agent.args }}
-          args: "{{ .Values.agent.args }}"
-          {{- else }}
-          args: "^${computer.jnlpmac} ^${computer.name}"
-          {{- end }}
+          args: "{{ .Values.agent.args | replace "$" "^$"}}"
```
Comment (wmcdona89) [inline stable/jenkins/templates/_helpers.tpl]: nit: add a space before the closing braces

Comment (torstenwalter) [reply stable/jenkins/templates/_helpers.tpl]: Thanks. Added it.

**Pull request merged**